pg_shard--1.0.sql

```diff
@@ -77,3 +77,78 @@ CREATE FUNCTION worker_copy_shard_placement(table_name text, source_node_name te
 RETURNS void
 AS 'MODULE_PATHNAME'
 LANGUAGE C STRICT;
+
+CREATE FUNCTION partition_column_to_node_string(table_oid oid)
+RETURNS text
+AS 'MODULE_PATHNAME'
+LANGUAGE C;
+
+COMMENT ON FUNCTION partition_column_to_node_string(oid)
+		IS 'return textual form of distributed table''s partition column';
+
+CREATE FUNCTION sync_table_metadata_to_citus(table_name text) RETURNS VOID
+AS $sync_table_metadata_to_citus$
+	DECLARE
+		table_relation_id CONSTANT oid NOT NULL := table_name::regclass::oid;
+		dummy_shard_length CONSTANT bigint := 0;
+	BEGIN
+		-- remove shard placements for table
+		DELETE FROM pg_dist_shard_placement
+		WHERE  shardid IN (SELECT id
+						   FROM   pgs_distribution_metadata.shard
+						   WHERE  relation_id = table_relation_id);
+
+		-- copy shard placement metadata
+		INSERT INTO pg_dist_shard_placement
+					(shardid,
+					 shardstate,
+					 shardlength,
+					 nodename,
+					 nodeport)
+		SELECT shard_id,
+			   shard_state,
+			   dummy_shard_length,
+			   node_name,
+			   node_port
+		FROM   pgs_distribution_metadata.shard_placement
+		WHERE  shard_id IN (SELECT id
+							FROM   pgs_distribution_metadata.shard
+							WHERE  relation_id = table_relation_id);
+
+		-- remove shards for table
+		DELETE FROM pg_dist_shard WHERE logicalrelid = table_relation_id;
+
+		-- copy shard metadata
+		INSERT INTO pg_dist_shard
+					(shardid,
+					 logicalrelid,
+					 shardstorage,
+					 shardminvalue,
+					 shardmaxvalue)
+		SELECT id,
+			   relation_id,
+			   STORAGE,
```

Comment:
Where is this STORAGE parameter coming from? I don't see it defined as a plpgsql variable?